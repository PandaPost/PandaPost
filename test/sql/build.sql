\set ECHO none

\i test/pgxntool/psql.sql

BEGIN;
CREATE EXTENSION IF NOT EXISTS plpythonu;

-- Tuple-only because of multiple calls to functions
\t
\i sql/panda_post.sql

\echo # TRANSACTION INTENTIONALLY LEFT OPEN!

-- vi: expandtab sw=2 ts=2
