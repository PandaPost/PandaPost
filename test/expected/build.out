\set ECHO none
 create_cast 
-------------
 
 
 
 
 
 
 
 
(8 rows)

# TRANSACTION INTENTIONALLY LEFT OPEN!
